test/integration/generated/datalakestorage/src/models/index.ts

```diff
@@ -82,7 +82,7 @@ export interface BlobItemInternal {
   deleted: boolean;
   snapshot: string;
   versionId?: string;
-  isCurrentVersion?: boolean;
```

Comment:
Not sure why we have this change

Reply:
look like this is also because Style.camel("IsCurrentVersion", true, {}, 3) gives us IsCurrentVersion. Really weird